### ISSUE TITLE
COntext is sometimes not changed for a existing blank tab when requested via right click

### ISSUE THREAD

**Issue by nekromoff** [mozilla/multi-account-containers#2474]
### Before submitting a bug report

- [X] I updated to the latest version of Multi-Account Container and tested if I can reproduce the issue
- [X] I searched for existing reports to see if it hasn't already been reported

### Step to reproduce

Right clicking on a blank tab and selecting new context/account sometimes does not change the context of that tab. I just had a situation, when I had to click on Open a new tab with the required context/account. That worked. However using existing blank tab wouldn't change the content, even when I tried to move it to a different context multiple times.

### Actual behavior

Tab in default context

### Expected behavior

Tab moved to requested context

### Additional informations

FF v108, v8.0.9

### Provide a copy of Troubleshooting Information page (optional)

```plain text
Application Basics
------------------

Name: Firefox
Version: 108.0
Build ID: 20221208122842
Distribution ID: canonical
User Agent: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:108.0) Gecko/20100101 Firefox/108.0
OS: Linux 5.15.0-53-generic #59-Ubuntu SMP Mon Oct 17 18:53:30 UTC 2022
OS Theme: Breeze / Adwaita
Multiprocess Windows: 1/1
Fission Windows: 1/1 Enabled by default
Remote Processes: 13
Enterprise Policies: Inactive
Google Location Service Key: Found
Google Safebrowsing Key: Found
Mozilla Location Service Key: Missing
Safe Mode: false
Memory Size (RAM): 30.8 GB
Disk Space Available: 627 GB

Crash Reports for the Last 3 Days
---------------------------------

Firefox Features
----------------

Name: Firefox Screenshots
Version: 39.0.1
ID: screenshots@mozilla.org

Name: Form Autofill
Version: 1.0.1
ID: formautofill@mozilla.org

Name: Picture-In-Picture
Version: 1.0.0
ID: pictureinpicture@mozilla.org

Name: Web Compatibility Interventions
Version: 108.1.0
ID: webcompat@mozilla.org

Name: WebCompat Reporter
Version: 1.5.1
ID: webcompat-reporter@mozilla.org

Remote Features
---------------

bug-1766618-rollout-disable-websocket-over-http2-release-91-100: active
bug-1802286-rollout-fix-webcompat-shim-breakage-release-103-107: active

Remote Processes
----------------

Type: Extension
Count: 1

Type: Privileged About
Count: 1

Type: Isolated Web Content
Count: 5

Type: Isolated Service Worker
Count: 2

Type: Preallocated
Count: 3

Type: Socket
Count: 1

Add-ons
-------

Name: Slovenská kontrola preklepov
Type: dictionary
Version: 2.4.6
Enabled: true
ID: sk@dictionaries.addons.mozilla.org

Name: Add-ons Search Detection
Type: extension
Version: 2.0.0
Enabled: true
ID: addons-search-detection@mozilla.com

Name: Amazon.com
Type: extension
Version: 1.4
Enabled: true
ID: amazondotcom@search.mozilla.org

Name: Bing
Type: extension
Version: 1.4
Enabled: true
ID: bing@search.mozilla.org

Name: Context Plus
Type: extension
Version: 0.6.0
Enabled: true
ID: {bb682c45-3136-4213-bf29-5f5833080bf4}

Name: Cookie Quick Manager
Type: extension
Version: 0.5rc2
Enabled: true
ID: {60f82f00-9ad5-4de5-b31c-b16a47c51558}

Name: D.Bridge 2
Type: extension
Version: 2.0.0.3
Enabled: true
ID: mozext@sk.ditec.dbridge2

Name: DuckDuckGo
Type: extension
Version: 1.2
Enabled: true
ID: ddg@search.mozilla.org

Name: Feedbro
Type: extension
Version: 4.14.1
Enabled: true
ID: {a9c2ad37-e940-4892-8dce-cd73c6cbbc0c}

Name: Firefox Multi-Account Containers
Type: extension
Version: 8.0.9
Enabled: true
ID: @testpilot-containers

Name: Google
Type: extension
Version: 1.3
Enabled: true
ID: google@search.mozilla.org

Name: HTTPS Everywhere
Type: extension
Version: 2021.7.13
Enabled: true
ID: https-everywhere@eff.org

Name: I don't care about cookies
Type: extension
Version: 3.4.4
Enabled: true
ID: jid1-KKzOGWgsW3Ao4Q@jetpack

Name: Laboratory
Type: extension
Version: 3.0.8
Enabled: true
ID: 1b2383b324c8520974ee097e46301d5ca4e076de387c02886f1c6b1503671586@pokeinthe.io

Name: LastPass: Free Password Manager
Type: extension
Version: 4.105.0.2
Enabled: true
ID: support@lastpass.com

Name: Privacy Badger
Type: extension
Version: 2022.9.27
Enabled: true
ID: jid1-MnnxcxisBPnSXQ@jetpack

Name: Temporary Containers
Type: extension
Version: 1.9.2
Enabled: true
ID: {c607c8df-14a7-4f28-894f-29e8722976af}

Name: uBlock Origin
Type: extension
Version: 1.45.2
Enabled: true
ID: uBlock0@raymondhill.net

Name: Wikipedia (en)
Type: extension
Version: 1.2
Enabled: true
ID: wikipedia@search.mozilla.org

Name: Yet Another Bitcoin Price Ticker
Type: extension
Version: 1.5.3
Enabled: true
ID: jid1-BYl3kts609WbCw@jetpack

Name: Español (AR) Language Pack
Type: locale
Version: 108.0buildid20221208.122842
Enabled: true
ID: langpack-es-AR@firefox.mozilla.org

Name: Español (Chile) Language Pack
Type: locale
Version: 108.0buildid20221208.122842
Enabled: true
ID: langpack-es-CL@firefox.mozilla.org

Name: Español (España) Language Pack
Type: locale
Version: 108.0buildid20221208.122842
Enabled: true
ID: langpack-es-ES@firefox.mozilla.org

Name: Español (México) Language Pack
Type: locale
Version: 108.0buildid20221208.122842
Enabled: true
ID: langpack-es-MX@firefox.mozilla.org

Name: Google Analytics Debugger
Type: extension
Version: 2.8.2
Enabled: false
ID: {ca6be013-d122-41e1-a41e-98092dfff1e3}

Graphics
--------

Features
Compositing: WebRender
Asynchronous Pan/Zoom: wheel input enabled; scrollbar drag enabled; keyboard enabled; autoscroll enabled; smooth pinch-zoom enabled
WebGL 1 Driver WSI Info: EGL_VENDOR: Mesa Project EGL_VERSION: 1.5 EGL_EXTENSIONS: EGL_ANDROID_blob_cache EGL_ANDROID_native_fence_sync EGL_CHROMIUM_sync_control EGL_EXT_buffer_age EGL_EXT_create_context_robustness EGL_EXT_image_dma_buf_import EGL_EXT_image_dma_buf_import_modifiers EGL_EXT_protected_surface EGL_EXT_swap_buffers_with_damage EGL_KHR_cl_event2 EGL_KHR_config_attribs EGL_KHR_create_context EGL_KHR_create_context_no_error EGL_KHR_fence_sync EGL_KHR_get_all_proc_addresses EGL_KHR_gl_colorspace EGL_KHR_gl_renderbuffer_image EGL_KHR_gl_texture_2D_image EGL_KHR_gl_texture_3D_image EGL_KHR_gl_texture_cubemap_image EGL_KHR_image EGL_KHR_image_base EGL_KHR_image_pixmap EGL_KHR_no_config_context EGL_KHR_reusable_sync EGL_KHR_surfaceless_context EGL_KHR_swap_buffers_with_damage EGL_EXT_pixel_format_float EGL_KHR_wait_sync EGL_MESA_configless_context EGL_MESA_drm_image EGL_MESA_image_dma_buf_export EGL_MESA_query_driver EGL_NOK_texture_from_pixmap EGL_WL_bind_wayland_display EGL_EXTENSIONS(nullptr): EGL_EXT_device_base EGL_EXT_device_enumeration EGL_EXT_device_query EGL_EXT_platform_base EGL_KHR_client_get_all_proc_addresses EGL_EXT_client_extensions EGL_KHR_debug EGL_EXT_platform_device EGL_EXT_platform_wayland EGL_KHR_platform_wayland EGL_EXT_platform_x11 EGL_KHR_platform_x11 EGL_MESA_platform_xcb EGL_MESA_platform_gbm EGL_KHR_platform_gbm EGL_MESA_platform_surfaceless IsWebglOutOfProcessEnabled: 1
WebGL 1 Driver Renderer: AMD -- AMD RENOIR (LLVM 13.0.1, DRM 3.42, 5.15.0-53-generic)
WebGL 1 Driver Version: 4.6 (Compatibility Profile) Mesa 22.0.5
WebGL 1 Driver Extensions: GL_ARB_multisample GL_EXT_abgr GL_EXT_bgra GL_EXT_blend_color GL_EXT_blend_minmax GL_EXT_blend_subtract GL_EXT_copy_texture GL_EXT_subtexture GL_EXT_texture_object GL_EXT_vertex_array GL_EXT_compiled_vertex_array GL_EXT_texture GL_EXT_texture3D GL_IBM_rasterpos_clip GL_ARB_point_parameters GL_EXT_draw_range_elements GL_EXT_packed_pixels GL_EXT_point_parameters GL_EXT_rescale_normal GL_EXT_separate_specular_color GL_EXT_texture_edge_clamp GL_SGIS_generate_mipmap GL_SGIS_texture_border_clamp GL_SGIS_texture_edge_clamp GL_SGIS_texture_lod GL_ARB_framebuffer_sRGB GL_ARB_multitexture GL_EXT_framebuffer_sRGB GL_IBM_multimode_draw_arrays GL_IBM_texture_mirrored_repeat GL_ARB_texture_cube_map GL_ARB_texture_env_add GL_ARB_transpose_matrix GL_EXT_blend_func_separate GL_EXT_fog_coord GL_EXT_multi_draw_arrays GL_EXT_secondary_color GL_EXT_texture_env_add GL_EXT_texture_filter_anisotropic GL_EXT_texture_lod_bias GL_INGR_blend_func_separate GL_NV_blend_square GL_NV_light_max_exponent GL_NV_texgen_reflection GL_NV_texture_env_combine4 GL_S3_s3tc GL_SUN_multi_draw_arrays GL_ARB_texture_border_clamp GL_ARB_texture_compression GL_EXT_framebuffer_object GL_EXT_texture_compression_s3tc GL_EXT_texture_env_combine GL_EXT_texture_env_dot3 GL_MESA_window_pos GL_NV_packed_depth_stencil GL_NV_texture_rectangle GL_ARB_depth_texture GL_ARB_occlusion_query GL_ARB_shadow GL_ARB_texture_env_combine GL_ARB_texture_env_crossbar GL_ARB_texture_env_dot3 GL_ARB_texture_mirrored_repeat GL_ARB_window_pos GL_ATI_fragment_shader GL_EXT_stencil_two_side GL_EXT_texture_cube_map GL_NV_copy_depth_to_color GL_NV_depth_clamp GL_NV_fog_distance GL_NV_half_float GL_APPLE_packed_pixels GL_ARB_draw_buffers GL_ARB_fragment_program GL_ARB_fragment_shader GL_ARB_shader_objects GL_ARB_vertex_program GL_ARB_vertex_shader GL_ATI_draw_buffers GL_ATI_texture_env_combine3 GL_ATI_texture_float GL_EXT_depth_bounds_test GL_EXT_shadow_funcs GL_EXT_stencil_wrap GL_MESA_pack_invert GL_NV_primitive_restart GL_ARB_depth_clamp GL_ARB_fragment_program_shadow GL_ARB_half_float_pixel GL_ARB_occlusion_query2 GL_ARB_point_sprite GL_ARB_shading_language_100 GL_ARB_sync GL_ARB_texture_non_power_of_two GL_ARB_vertex_buffer_object GL_ATI_blend_equation_separate GL_EXT_blend_equation_separate GL_OES_read_format GL_ARB_color_buffer_float GL_ARB_pixel_buffer_object GL_ARB_texture_compression_rgtc GL_ARB_texture_float GL_ARB_texture_rectangle GL_ATI_texture_compression_3dc GL_EXT_packed_float GL_EXT_pixel_buffer_object GL_EXT_texture_compression_dxt1 GL_EXT_texture_compression_rgtc GL_EXT_texture_mirror_clamp GL_EXT_texture_rectangle GL_EXT_texture_sRGB GL_EXT_texture_shared_exponent GL_ARB_framebuffer_object GL_EXT_framebuffer_blit GL_EXT_framebuffer_multisample GL_EXT_packed_depth_stencil GL_ARB_vertex_array_object GL_ATI_separate_stencil GL_ATI_texture_mirror_once GL_EXT_draw_buffers2 GL_EXT_draw_instanced GL_EXT_gpu_program_parameters GL_EXT_gpu_shader4 GL_EXT_texture_array GL_EXT_texture_compression_latc GL_EXT_texture_integer GL_EXT_texture_sRGB_decode GL_EXT_timer_query GL_OES_EGL_image GL_AMD_performance_monitor GL_EXT_texture_buffer_object GL_AMD_texture_texture4 GL_ARB_copy_buffer GL_ARB_depth_buffer_float GL_ARB_draw_instanced GL_ARB_half_float_vertex GL_ARB_instanced_arrays GL_ARB_map_buffer_range GL_ARB_texture_buffer_object GL_ARB_texture_rg GL_ARB_texture_swizzle GL_ARB_vertex_array_bgra GL_EXT_texture_swizzle GL_EXT_vertex_array_bgra GL_NV_conditional_render GL_AMD_conservative_depth GL_AMD_depth_clamp_separate GL_AMD_draw_buffers_blend GL_AMD_seamless_cubemap_per_texture GL_AMD_shader_stencil_export GL_ARB_ES2_compatibility GL_ARB_blend_func_extended GL_ARB_compatibility GL_ARB_debug_output GL_ARB_draw_buffers_blend GL_ARB_draw_elements_base_vertex GL_ARB_explicit_attrib_location GL_ARB_fragment_coord_conventions GL_ARB_provoking_vertex GL_ARB_sample_shading GL_ARB_sampler_objects GL_ARB_seamless_cube_map GL_ARB_shader_stencil_export GL_ARB_shader_texture_lod GL_ARB_tessellation_shader GL_ARB_texture_buffer_object_rgb32 GL_ARB_texture_cube_map_array GL_ARB_texture_gather GL_ARB_texture_multisample GL_ARB_texture_query_lod GL_ARB_texture_rgb10_a2ui GL_ARB_uniform_buffer_object GL_ARB_vertex_type_2_10_10_10_rev GL_ATI_meminfo GL_EXT_provoking_vertex GL_EXT_texture_snorm GL_MESA_texture_signed_rgba GL_NV_copy_image GL_NV_texture_barrier GL_ARB_draw_indirect GL_ARB_get_program_binary GL_ARB_gpu_shader5 GL_ARB_gpu_shader_fp64 GL_ARB_robustness GL_ARB_separate_shader_objects GL_ARB_shader_bit_encoding GL_ARB_shader_precision GL_ARB_shader_subroutine GL_ARB_texture_compression_bptc GL_ARB_timer_query GL_ARB_transform_feedback2 GL_ARB_transform_feedback3 GL_ARB_vertex_attrib_64bit GL_ARB_viewport_array GL_EXT_direct_state_access GL_EXT_shader_image_load_store GL_EXT_vertex_attrib_64bit GL_NV_vdpau_interop GL_AMD_multi_draw_indirect GL_ANGLE_texture_compression_dxt3 GL_ANGLE_texture_compression_dxt5 GL_ARB_base_instance GL_ARB_compressed_texture_pixel_storage GL_ARB_conservative_depth GL_ARB_internalformat_query GL_ARB_map_buffer_alignment GL_ARB_shader_atomic_counters GL_ARB_shader_image_load_store GL_ARB_shading_language_420pack GL_ARB_shading_language_packing GL_ARB_texture_storage GL_ARB_transform_feedback_instanced GL_EXT_framebuffer_multisample_blit_scaled GL_EXT_transform_feedback GL_AMD_query_buffer_object GL_AMD_shader_trinary_minmax GL_AMD_vertex_shader_layer GL_AMD_vertex_shader_viewport_index GL_ARB_ES3_compatibility GL_ARB_arrays_of_arrays GL_ARB_clear_buffer_object GL_ARB_compute_shader GL_ARB_copy_image GL_ARB_explicit_uniform_location GL_ARB_fragment_layer_viewport GL_ARB_framebuffer_no_attachments GL_ARB_invalidate_subdata GL_ARB_multi_draw_indirect GL_ARB_program_interface_query GL_ARB_robust_buffer_access_behavior GL_ARB_shader_image_size GL_ARB_shader_storage_buffer_object GL_ARB_stencil_texturing GL_ARB_texture_buffer_range GL_ARB_texture_query_levels GL_ARB_texture_storage_multisample GL_ARB_texture_view GL_ARB_vertex_attrib_binding GL_KHR_debug GL_KHR_robustness GL_KHR_texture_compression_astc_ldr GL_AMD_pinned_memory GL_ARB_bindless_texture GL_ARB_buffer_storage GL_ARB_clear_texture GL_ARB_compute_variable_group_size GL_ARB_enhanced_layouts GL_ARB_indirect_parameters GL_ARB_internalformat_query2 GL_ARB_multi_bind GL_ARB_query_buffer_object GL_ARB_seamless_cubemap_per_texture GL_ARB_shader_draw_parameters GL_ARB_shader_group_vote GL_ARB_shading_language_include GL_ARB_sparse_texture GL_ARB_texture_mirror_clamp_to_edge GL_ARB_texture_stencil8 GL_ARB_vertex_type_10f_11f_11f_rev GL_EXT_shader_integer_mix GL_NVX_gpu_memory_info GL_ARB_ES3_1_compatibility GL_ARB_clip_control GL_ARB_conditional_render_inverted GL_ARB_cull_distance GL_ARB_derivative_control GL_ARB_direct_state_access GL_ARB_get_texture_sub_image GL_ARB_pipeline_statistics_query GL_ARB_shader_texture_image_samples GL_ARB_sparse_buffer GL_ARB_texture_barrier GL_ARB_transform_feedback_overflow_query GL_EXT_polygon_offset_clamp GL_EXT_shader_image_load_formatted GL_KHR_blend_equation_advanced GL_KHR_context_flush_control GL_KHR_robust_buffer_access_behavior GL_NV_shader_atomic_int64 GL_ARB_ES3_2_compatibility GL_ARB_gpu_shader_int64 GL_ARB_parallel_shader_compile GL_ARB_shader_atomic_counter_ops GL_ARB_shader_ballot GL_ARB_shader_clock GL_ARB_shader_viewport_layer_array GL_ARB_sparse_texture2 GL_ARB_sparse_texture_clamp GL_EXT_shader_samples_identical GL_EXT_texture_sRGB_R8 GL_KHR_no_error GL_KHR_texture_compression_astc_sliced_3d GL_ARB_gl_spirv GL_ARB_spirv_extensions GL_EXT_window_rectangles GL_MESA_shader_integer_functions GL_ARB_polygon_offset_clamp GL_ARB_texture_filter_anisotropic GL_EXT_memory_object GL_EXT_memory_object_fd GL_EXT_semaphore GL_EXT_semaphore_fd GL_KHR_parallel_shader_compile GL_NV_alpha_to_coverage_dither_control GL_AMD_framebuffer_multisample_advanced GL_EXT_EGL_image_storage GL_EXT_texture_shadow_lod GL_INTEL_blackhole_render GL_MESA_framebuffer_flip_y GL_NV_compute_shader_derivatives GL_EXT_EGL_sync GL_EXT_demote_to_helper_invocation
WebGL 1 Extensions: ANGLE_instanced_arrays EXT_blend_minmax EXT_color_buffer_half_float EXT_float_blend EXT_frag_depth EXT_shader_texture_lod EXT_sRGB EXT_texture_compression_bptc EXT_texture_compression_rgtc EXT_texture_filter_anisotropic MOZ_debug OES_element_index_uint OES_fbo_render_mipmap OES_standard_derivatives OES_texture_float OES_texture_float_linear OES_texture_half_float OES_texture_half_float_linear OES_vertex_array_object WEBGL_color_buffer_float WEBGL_compressed_texture_astc WEBGL_compressed_texture_etc WEBGL_compressed_texture_s3tc WEBGL_compressed_texture_s3tc_srgb WEBGL_debug_renderer_info WEBGL_debug_shaders WEBGL_depth_texture WEBGL_draw_buffers WEBGL_lose_context
WebGL 2 Driver WSI Info: EGL_VENDOR: Mesa Project EGL_VERSION: 1.5 EGL_EXTENSIONS: EGL_ANDROID_blob_cache EGL_ANDROID_native_fence_sync EGL_CHROMIUM_sync_control EGL_EXT_buffer_age EGL_EXT_create_context_robustness EGL_EXT_image_dma_buf_import EGL_EXT_image_dma_buf_import_modifiers EGL_EXT_protected_surface EGL_EXT_swap_buffers_with_damage EGL_KHR_cl_event2 EGL_KHR_config_attribs EGL_KHR_create_context EGL_KHR_create_context_no_error EGL_KHR_fence_sync EGL_KHR_get_all_proc_addresses EGL_KHR_gl_colorspace EGL_KHR_gl_renderbuffer_image EGL_KHR_gl_texture_2D_image EGL_KHR_gl_texture_3D_image EGL_KHR_gl_texture_cubemap_image EGL_KHR_image EGL_KHR_image_base EGL_KHR_image_pixmap EGL_KHR_no_config_context EGL_KHR_reusable_sync EGL_KHR_surfaceless_context EGL_KHR_swap_buffers_with_damage EGL_EXT_pixel_format_float EGL_KHR_wait_sync EGL_MESA_configless_context EGL_MESA_drm_image EGL_MESA_image_dma_buf_export EGL_MESA_query_driver EGL_NOK_texture_from_pixmap EGL_WL_bind_wayland_display EGL_EXTENSIONS(nullptr): EGL_EXT_device_base EGL_EXT_device_enumeration EGL_EXT_device_query EGL_EXT_platform_base EGL_KHR_client_get_all_proc_addresses EGL_EXT_client_extensions EGL_KHR_debug EGL_EXT_platform_device EGL_EXT_platform_wayland EGL_KHR_platform_wayland EGL_EXT_platform_x11 EGL_KHR_platform_x11 EGL_MESA_platform_xcb EGL_MESA_platform_gbm EGL_KHR_platform_gbm EGL_MESA_platform_surfaceless IsWebglOutOfProcessEnabled: 1
WebGL 2 Driver Renderer: AMD -- AMD RENOIR (LLVM 13.0.1, DRM 3.42, 5.15.0-53-generic)
WebGL 2 Driver Version: 4.6 (Core Profile) Mesa 22.0.5
WebGL 2 Driver Extensions: GL_AMD_conservative_depth GL_AMD_depth_clamp_separate GL_AMD_draw_buffers_blend GL_AMD_framebuffer_multisample_advanced GL_AMD_gpu_shader_int64 GL_AMD_multi_draw_indirect GL_AMD_performance_monitor GL_AMD_pinned_memory GL_AMD_query_buffer_object GL_AMD_seamless_cubemap_per_texture GL_AMD_shader_stencil_export GL_AMD_shader_trinary_minmax GL_AMD_texture_texture4 GL_AMD_vertex_shader_layer GL_AMD_vertex_shader_viewport_index GL_ANGLE_texture_compression_dxt3 GL_ANGLE_texture_compression_dxt5 GL_ARB_ES2_compatibility GL_ARB_ES3_1_compatibility GL_ARB_ES3_2_compatibility GL_ARB_ES3_compatibility GL_ARB_arrays_of_arrays GL_ARB_base_instance GL_ARB_bindless_texture GL_ARB_blend_func_extended GL_ARB_buffer_storage GL_ARB_clear_buffer_object GL_ARB_clear_texture GL_ARB_clip_control GL_ARB_color_buffer_float GL_ARB_compressed_texture_pixel_storage GL_ARB_compute_shader GL_ARB_compute_variable_group_size GL_ARB_conditional_render_inverted GL_ARB_conservative_depth GL_ARB_copy_buffer GL_ARB_copy_image GL_ARB_cull_distance GL_ARB_debug_output GL_ARB_depth_buffer_float GL_ARB_depth_clamp GL_ARB_derivative_control GL_ARB_direct_state_access GL_ARB_draw_buffers GL_ARB_draw_buffers_blend GL_ARB_draw_elements_base_vertex GL_ARB_draw_indirect GL_ARB_draw_instanced GL_ARB_enhanced_layouts GL_ARB_explicit_attrib_location GL_ARB_explicit_uniform_location GL_ARB_fragment_coord_conventions GL_ARB_fragment_layer_viewport GL_ARB_fragment_shader GL_ARB_framebuffer_no_attachments GL_ARB_framebuffer_object GL_ARB_framebuffer_sRGB GL_ARB_get_program_binary GL_ARB_get_texture_sub_image GL_ARB_gl_spirv GL_ARB_gpu_shader5 GL_ARB_gpu_shader_fp64 GL_ARB_gpu_shader_int64 GL_ARB_half_float_pixel GL_ARB_half_float_vertex GL_ARB_indirect_parameters GL_ARB_instanced_arrays GL_ARB_internalformat_query GL_ARB_internalformat_query2 GL_ARB_invalidate_subdata GL_ARB_map_buffer_alignment GL_ARB_map_buffer_range GL_ARB_multi_bind GL_ARB_multi_draw_indirect GL_ARB_occlusion_query2 GL_ARB_parallel_shader_compile GL_ARB_pipeline_statistics_query GL_ARB_pixel_buffer_object GL_ARB_point_sprite GL_ARB_polygon_offset_clamp GL_ARB_program_interface_query GL_ARB_provoking_vertex GL_ARB_query_buffer_object GL_ARB_robust_buffer_access_behavior GL_ARB_robustness GL_ARB_sample_shading GL_ARB_sampler_objects GL_ARB_seamless_cube_map GL_ARB_seamless_cubemap_per_texture GL_ARB_separate_shader_objects GL_ARB_shader_atomic_counter_ops GL_ARB_shader_atomic_counters GL_ARB_shader_ballot GL_ARB_shader_bit_encoding GL_ARB_shader_clock GL_ARB_shader_draw_parameters GL_ARB_shader_group_vote GL_ARB_shader_image_load_store GL_ARB_shader_image_size GL_ARB_shader_objects GL_ARB_shader_precision GL_ARB_shader_stencil_export GL_ARB_shader_storage_buffer_object GL_ARB_shader_subroutine GL_ARB_shader_texture_image_samples GL_ARB_shader_texture_lod GL_ARB_shader_viewport_layer_array GL_ARB_shading_language_420pack GL_ARB_shading_language_include GL_ARB_shading_language_packing GL_ARB_sparse_buffer GL_ARB_sparse_texture GL_ARB_sparse_texture2 GL_ARB_sparse_texture_clamp GL_ARB_spirv_extensions GL_ARB_stencil_texturing GL_ARB_sync GL_ARB_tessellation_shader GL_ARB_texture_barrier GL_ARB_texture_buffer_object GL_ARB_texture_buffer_object_rgb32 GL_ARB_texture_buffer_range GL_ARB_texture_compression_bptc GL_ARB_texture_compression_rgtc GL_ARB_texture_cube_map_array GL_ARB_texture_filter_anisotropic GL_ARB_texture_float GL_ARB_texture_gather GL_ARB_texture_mirror_clamp_to_edge GL_ARB_texture_multisample GL_ARB_texture_non_power_of_two GL_ARB_texture_query_levels GL_ARB_texture_query_lod GL_ARB_texture_rectangle GL_ARB_texture_rg GL_ARB_texture_rgb10_a2ui GL_ARB_texture_stencil8 GL_ARB_texture_storage GL_ARB_texture_storage_multisample GL_ARB_texture_swizzle GL_ARB_texture_view GL_ARB_timer_query GL_ARB_transform_feedback2 GL_ARB_transform_feedback3 GL_ARB_transform_feedback_instanced GL_ARB_transform_feedback_overflow_query GL_ARB_uniform_buffer_object GL_ARB_vertex_array_bgra GL_ARB_vertex_array_object GL_ARB_vertex_attrib_64bit GL_ARB_vertex_attrib_binding GL_ARB_vertex_buffer_object GL_ARB_vertex_shader GL_ARB_vertex_type_10f_11f_11f_rev GL_ARB_vertex_type_2_10_10_10_rev GL_ARB_viewport_array GL_ATI_blend_equation_separate GL_ATI_meminfo GL_ATI_texture_float GL_ATI_texture_mirror_once GL_EXT_EGL_image_storage GL_EXT_EGL_sync GL_EXT_abgr GL_EXT_blend_equation_separate GL_EXT_demote_to_helper_invocation GL_EXT_depth_bounds_test GL_EXT_draw_buffers2 GL_EXT_draw_instanced GL_EXT_framebuffer_blit GL_EXT_framebuffer_multisample GL_EXT_framebuffer_multisample_blit_scaled GL_EXT_framebuffer_object GL_EXT_framebuffer_sRGB GL_EXT_memory_object GL_EXT_memory_object_fd GL_EXT_packed_depth_stencil GL_EXT_packed_float GL_EXT_pixel_buffer_object GL_EXT_polygon_offset_clamp GL_EXT_provoking_vertex GL_EXT_semaphore GL_EXT_semaphore_fd GL_EXT_shader_image_load_formatted GL_EXT_shader_image_load_store GL_EXT_shader_integer_mix GL_EXT_shader_samples_identical GL_EXT_texture_array GL_EXT_texture_compression_dxt1 GL_EXT_texture_compression_rgtc GL_EXT_texture_compression_s3tc GL_EXT_texture_filter_anisotropic GL_EXT_texture_integer GL_EXT_texture_mirror_clamp GL_EXT_texture_sRGB GL_EXT_texture_sRGB_R8 GL_EXT_texture_sRGB_decode GL_EXT_texture_shadow_lod GL_EXT_texture_shared_exponent GL_EXT_texture_snorm GL_EXT_texture_swizzle GL_EXT_timer_query GL_EXT_transform_feedback GL_EXT_vertex_array_bgra GL_EXT_vertex_attrib_64bit GL_EXT_window_rectangles GL_IBM_multimode_draw_arrays GL_INTEL_blackhole_render GL_KHR_blend_equation_advanced GL_KHR_context_flush_control GL_KHR_debug GL_KHR_no_error GL_KHR_parallel_shader_compile GL_KHR_robust_buffer_access_behavior GL_KHR_robustness GL_KHR_texture_compression_astc_ldr GL_KHR_texture_compression_astc_sliced_3d GL_MESA_framebuffer_flip_y GL_MESA_pack_invert GL_MESA_shader_integer_functions GL_MESA_texture_signed_rgba GL_NVX_gpu_memory_info GL_NV_alpha_to_coverage_dither_control GL_NV_compute_shader_derivatives GL_NV_conditional_render GL_NV_copy_image GL_NV_depth_clamp GL_NV_packed_depth_stencil GL_NV_shader_atomic_int64 GL_NV_texture_barrier GL_NV_vdpau_interop GL_OES_EGL_image GL_S3_s3tc
WebGL 2 Extensions: EXT_color_buffer_float EXT_float_blend EXT_texture_compression_bptc EXT_texture_compression_rgtc EXT_texture_filter_anisotropic MOZ_debug OES_draw_buffers_indexed OES_texture_float_linear WEBGL_compressed_texture_astc WEBGL_compressed_texture_etc WEBGL_compressed_texture_s3tc WEBGL_compressed_texture_s3tc_srgb WEBGL_debug_renderer_info WEBGL_debug_shaders WEBGL_lose_context
Window Protocol: x11
Desktop Environment: kde
Target Frame Rate: 60
GPU #1
Active: Yes
Description: AMD RENOIR (LLVM 13.0.1, DRM 3.42, 5.15.0-53-generic)
Vendor ID: 0x1002
Device ID: 0x1636
Driver Vendor: mesa/radeonsi
Driver Version: 22.0.5.0
RAM: 0

Diagnostics
AzureCanvasBackend: skia
AzureContentBackend: skia
AzureFallbackCanvasBackend: skia
CMSOutputProfile: Empty profile data
Display0: 1920x1080@60Hz scales:1.000000|1.000000
DisplayCount: 1
Decision Log
HW_COMPOSITING:
available by default
OPENGL_COMPOSITING:
available by default
WEBRENDER:
available by default
force_enabled by user: Force enabled by pref
WEBRENDER_COMPOSITOR:
disabled by default: Disabled by default
blocklisted by env: Blocklisted by gfxInfo
blocked by runtime: Cannot be enabled in release or beta
WEBRENDER_PARTIAL:
available by default
WEBRENDER_SHADER_CACHE:
disabled by default: Disabled by default
WEBRENDER_OPTIMIZED_SHADERS:
available by default
WEBRENDER_ANGLE:
available by default
unavailable by env: OS not supported
WEBRENDER_DCOMP_PRESENT:
available by default
disabled by user: User disabled via pref
unavailable by env: Requires Windows 10 or later
unavailable by runtime: Requires ANGLE
WEBGPU:
disabled by default: Disabled by default
blocked by runtime: WebGPU cannot be enabled in release or beta
X11_EGL:
available by default
DMABUF:
available by default
HARDWARE_VIDEO_DECODING:
available by default
blocklisted by env: Blocklisted by gfxInfo
DMABUF_SURFACE_EXPORT:
blocked by default: Blocklisted by gfxInfo
BACKDROP_FILTER:
available by default




Media
-----

Audio Backend: pulse-rust
Max Channels: 2
Preferred Sample Rate: 44100
Roundtrip latency (standard deviation): ...
Output Devices
Name: Group
Family 17h (Models 10h-1fh) HD Audio Controller Analog Stereo: /devices/pci0000:00/0000:00:08.1/0000:04:00.6/sound/card1
Built-in Audio Analog Stereo: /devices/platform/snd_aloop.0/sound/card2
Renoir Radeon High Definition Audio Controller Digital Stereo (HDMI): /devices/pci0000:00/0000:00:08.1/0000:04:00.1/sound/card0
Input Devices
Name: Group
Monitor of Family 17h (Models 10h-1fh) HD Audio Controller Analog Stereo: /devices/pci0000:00/0000:00:08.1/0000:04:00.6/sound/card1
Family 17h (Models 10h-1fh) HD Audio Controller Analog Stereo: /devices/pci0000:00/0000:00:08.1/0000:04:00.6/sound/card1
Monitor of Built-in Audio Analog Stereo: /devices/platform/snd_aloop.0/sound/card2
Built-in Audio Analog Stereo: /devices/platform/snd_aloop.0/sound/card2
Monitor of Renoir Radeon High Definition Audio Controller Digital Stereo (HDMI): /devices/pci0000:00/0000:00:08.1/0000:04:00.1/sound/card0

Enumerate database

Environment Variables
---------------------

DISPLAY: :0
MOZ_USE_XINPUT2: 1
MOZ_APP_LAUNCHER: /usr/bin/firefox
MOZ_ASSUME_USER_NS: 1
MOZ_CRASHREPORTER_EVENTS_DIRECTORY: /home/dusoft/.mozilla/firefox/emol25ca.default-release/crashes/events
MOZ_CRASHREPORTER_RESTART_ARG_0: /usr/bin/firefox
MOZ_CRASHREPORTER_RESTART_ARG_1: --sm-client-id
MOZ_CRASHREPORTER_RESTART_ARG_2: 101a51c1178144000166938869500000027890074
MOZ_CRASHREPORTER_RESTART_ARG_3:
MOZ_CRASHREPORTER_DATA_DIRECTORY: /home/dusoft/.mozilla/firefox/Crash Reports
MOZ_CRASHREPORTER_PING_DIRECTORY: /home/dusoft/.mozilla/firefox/Pending Pings
MOZ_CRASHREPORTER_STRINGS_OVERRIDE: /usr/lib/firefox/browser/crashreporter-override.ini
MOZ_LAUNCHED_CHILD:
MOZ_APP_SILENT_START:
XRE_PROFILE_PATH:
XRE_PROFILE_LOCAL_PATH:
XRE_START_OFFLINE:
XRE_BINARY_PATH:
XRE_RESTARTED_BY_PROFILE_MANAGER:

Experimental Features
---------------------

about:home startup cache (browser.startup.homepage.abouthome_cache.enabled): false
Accessibility cache (accessibility.cache.enabled): false
Cookies: SameSite=Lax by default (network.cookie.sameSite.laxByDefault): false
Cookies: SameSite=None requires secure attribute (network.cookie.sameSite.noneRequiresSecure): false
CSS: Masonry Layout (layout.css.grid-template-masonry-value.enabled): false
Developer Tools: Compatibility Panel (devtools.inspector.compatibility.enabled): true
Developer Tools: Service Worker debugging (devtools.debugger.features.windowless-service-workers): false
Media: JPEG XL (image.jxl.enabled): false
Address Bar: show results during IME composition (browser.urlbar.keepPanelOpenDuringImeComposition): false
Web API: WebGPU (dom.webgpu.enabled): false
WebRTC Global Mute Toggles (privacy.webrtc.globalMuteToggles): false

Remote Experiments
------------------

about:home startup cache 3: treatment-a
Task Continuity - Sync after tab change Rollout 60%: sync-after-tab

Important Modified Preferences
------------------------------

accessibility.typeaheadfind.flashBar: 0
browser.contentblocking.category: strict
browser.search.region: SK
browser.sessionstore.upgradeBackup.latestBuildID: 20221208122842
browser.startup.homepage: https://www.google.com/
browser.startup.homepage_override.buildID: 20221208122842
browser.startup.homepage_override.mstone: 108.0
browser.tabs.firefox-view.ui-state.recently-closed-tabs.open: false
browser.tabs.firefox-view.ui-state.tab-pickup.open: true
browser.urlbar.placeholderName: Google
browser.urlbar.placeholderName.private: Google
browser.urlbar.quicksuggest.migrationVersion: 2
browser.urlbar.quicksuggest.scenario: history
browser.urlbar.resultBuckets: {"children":[{"maxResultCount":1,"children":[{"group":"heuristicTest"},{"group":"heuristicExtension"},{"group":"heuristi
browser.urlbar.resultGroups: {"children":[{"maxResultCount":1,"children":[{"group":"heuristicTest"},{"group":"heuristicExtension"},{"group":"heuristi
browser.urlbar.shortcuts.tabs: false
browser.urlbar.showSearchSuggestionsFirst: false
browser.urlbar.tabToSearch.onboard.interactionsLeft: 0
browser.urlbar.tipShownCount.searchTip_onboard: 4
browser.urlbar.tipShownCount.searchTip_redirect: 4
browser.urlbar.tipShownCount.tabToSearch: 60
doh-rollout.balrog-migration-done: true
doh-rollout.doneFirstRun: true
doh-rollout.home-region: SK
extensions.formautofill.addresses.usage.hasEntry: false
extensions.lastAppVersion: 108.0
font.internaluseonly.changed: false
gfx.blacklist.layers.opengl.failureid: FEATURE_FAILURE_SOFTWARE_GL
gfx.webrender.all: true
gfx.webrender.enabled: true
idle.lastDailyNotification: 1670966084
media.eme.enabled: true
media.gmp-gmpopenh264.abi: x86_64-gcc3
media.gmp-gmpopenh264.lastDownloadFailed: 1666449439
media.gmp-gmpopenh264.lastDownloadFailReason: Error: Failed downloading via ServiceRequest, status: 0, reason: error
media.gmp-gmpopenh264.lastInstallStart: 1666449431
media.gmp-gmpopenh264.lastUpdate: 1655646022
media.gmp-gmpopenh264.version: 1.8.1.2
media.gmp-manager.buildID: 20221208122842
media.gmp-manager.lastCheck: 1670762614
media.gmp-manager.lastEmptyCheck: 1670762614
media.gmp-widevinecdm.abi: x86_64-gcc3
media.gmp-widevinecdm.lastDownload: 1669888273
media.gmp-widevinecdm.lastInstallStart: 1669888273
media.gmp-widevinecdm.lastUpdate: 1669888273
media.gmp-widevinecdm.version: 4.10.2557.0
media.gmp.storage.version.observed: 1
media.videocontrols.picture-in-picture.video-toggle.enabled: false
media.videocontrols.picture-in-picture.video-toggle.has-used: true
network.dns.disablePrefetch: true
network.http.referer.disallowCrossSiteRelaxingDefault.top_navigation: true
network.http.speculative-parallel-limit: 0
network.predictor.enabled: false
network.prefetch-next: false
network.trr.blocklist_cleanup_done: true
places.database.lastMaintenance: 1670966089
places.history.expiration.transient_current_max_pages: 131952
privacy.annotate_channels.strict_list.enabled: true
privacy.cpd.offlineApps: true
privacy.cpd.sessions: false
privacy.cpd.siteSettings: true
privacy.donottrackheader.enabled: true
privacy.partition.network_state.ocsp_cache: true
privacy.purge_trackers.date_in_cookie_database: 0
privacy.purge_trackers.last_purge: 1670966085326
privacy.query_stripping.enabled: true
privacy.query_stripping.e­nabled: true
privacy.query_stripping.enabled.pbmode: true
privacy.query_stripping.e­nabled.pbmode: true
privacy.restrict3rdpartystorage.rollout.enabledByDefault: true
privacy.sanitize.pending: [{"id":"newtab-container","itemsToClear":[],"options":{}}]
privacy.sanitize.timeSpan: 4
privacy.trackingprotection.enabled: true
privacy.trackingprotection.socialtracking.enabled: true
privacy.userContext.enabled: true
privacy.userContext.extension: {c607c8df-14a7-4f28-894f-29e8722976af}
privacy.userContext.ui.enabled: true
security.remote_settings.crlite_filters.checked: 1657358087
security.remote_settings.intermediates.checked: 1657358087
security.sandbox.content.tempDirSuffix: 27246339-91cd-43d3-b3b9-12ead02a0227
security.sandbox.plugin.tempDirSuffix: 9d1ffd0b-c1d6-4457-8f00-191321570c52
services.sync.declinedEngines: adblockplus,creditcards,forms,history,passwords
services.sync.engine.history: false
services.sync.engine.passwords: false
services.sync.engine.prefs.modified: false
services.sync.lastPing: 1670928734
services.sync.lastSync: Wed Dec 14 2022 09:56:41 GMT+0100 (Central European Standard Time)
signon.rememberSignons: false
signon.usage.hasEntry: true
storage.vacuum.last.index: 1
storage.vacuum.last.places.sqlite: 1668869391
webgl.force-enabled: true

Important Locked Preferences
----------------------------

fission.autostart.session: true

Places Database
---------------

Entity: sqlite_schema
Count: -
Size (KiB): 32
Size (%): 0
Efficiency (%): 30.4
Sequentiality (%): 37.8

Entity: moz_origins
Count: 8668
Size (KiB): 384
Size (%): 0.5
Efficiency (%): 79.5
Sequentiality (%): 37.8

Entity: sqlite_autoindex_moz_origins_1
Count: -
Size (KiB): 352
Size (%): 0.5
Efficiency (%): 80.4
Sequentiality (%): 37.8

Entity: moz_places
Count: 137456
Size (KiB): 38496
Size (%): 49.5
Efficiency (%): 94.5
Sequentiality (%): 37.8

Entity: moz_historyvisits
Count: 252191
Size (KiB): 6720
Size (%): 8.6
Efficiency (%): 98.3
Sequentiality (%): 37.8

Entity: moz_inputhistory
Count: 806
Size (KiB): 32
Size (%): 0
Efficiency (%): 70.8
Sequentiality (%): 37.8

Entity: sqlite_autoindex_moz_inputhistory_1
Count: -
Size (KiB): 32
Size (%): 0
Efficiency (%): 51.1
Sequentiality (%): 37.8

Entity: moz_bookmarks
Count: 576
Size (KiB): 96
Size (%): 0.1
Efficiency (%): 49.8
Sequentiality (%): 37.8

Entity: moz_bookmarks_deleted
Count: 0
Size (KiB): 32
Size (%): 0
Efficiency (%): 0
Sequentiality (%): 37.8

Entity: sqlite_autoindex_moz_bookmarks_deleted_1
Count: -
Size (KiB): 32
Size (%): 0
Efficiency (%): 0
Sequentiality (%): 37.8

Entity: moz_keywords
Count: 0
Size (KiB): 32
Size (%): 0
Efficiency (%): 0
Sequentiality (%): 37.8

Entity: sqlite_autoindex_moz_keywords_1
Count: -
Size (KiB): 32
Size (%): 0
Efficiency (%): 0
Sequentiality (%): 37.8

Entity: sqlite_sequence
Count: -
Size (KiB): 32
Size (%): 0
Efficiency (%): 0.1
Sequentiality (%): 37.8

Entity: moz_anno_attributes
Count: 2
Size (KiB): 32
Size (%): 0
Efficiency (%): 0.2
Sequentiality (%): 37.8

Entity: sqlite_autoindex_moz_anno_attributes_1
Count: -
Size (KiB): 32
Size (%): 0
Efficiency (%): 0.2
Sequentiality (%): 37.8

Entity: moz_annos
Count: 148
Size (KiB): 32
Size (%): 0
Efficiency (%): 41.5
Sequentiality (%): 37.8

Entity: moz_items_annos
Count: 0
Size (KiB): 32
Size (%): 0
Efficiency (%): 0
Sequentiality (%): 37.8

Entity: moz_meta
Count: 8
Size (KiB): 32
Size (%): 0
Efficiency (%): 1.1
Sequentiality (%): 37.8

Entity: sqlite_stat1
Count: -
Size (KiB): 32
Size (%): 0
Efficiency (%): 2.9
Sequentiality (%): 37.8

Entity: moz_places_metadata
Count: 0
Size (KiB): 32
Size (%): 0
Efficiency (%): 0
Sequentiality (%): 37.8

Entity: moz_places_metadata_search_queries
Count: 0
Size (KiB): 32
Size (%): 0
Efficiency (%): 0
Sequentiality (%): 37.8

Entity: sqlite_autoindex_moz_places_metadata_search_queries_1
Count: -
Size (KiB): 32
Size (%): 0
Efficiency (%): 0
Sequentiality (%): 37.8

Entity: moz_places_metadata_snapshots
Count: 0
Size (KiB): 32
Size (%): 0
Efficiency (%): 0
Sequentiality (%): 37.8

Entity: moz_places_metadata_snapshots_extra
Count: 0
Size (KiB): 32
Size (%): 0
Efficiency (%): 0
Sequentiality (%): 37.8

Entity: moz_places_metadata_snapshots_groups
Count: 0
Size (KiB): 32
Size (%): 0
Efficiency (%): 0
Sequentiality (%): 37.8

Entity: moz_places_metadata_groups_to_snapshots
Count: 0
Size (KiB): 32
Size (%): 0
Efficiency (%): 0
Sequentiality (%): 37.8

Entity: moz_session_metadata
Count: 0
Size (KiB): 32
Size (%): 0
Efficiency (%): 0
Sequentiality (%): 37.8

Entity: sqlite_autoindex_moz_session_metadata_1
Count: -
Size (KiB): 32
Size (%): 0
Efficiency (%): 0
Sequentiality (%): 37.8

Entity: moz_session_to_places
Count: 0
Size (KiB): 32
Size (%): 0
Efficiency (%): 0
Sequentiality (%): 37.8

Entity: moz_previews_tombstones
Count: 0
Size (KiB): 32
Size (%): 0
Efficiency (%): 0
Sequentiality (%): 37.8

Entity: moz_places_url_hashindex
Count: -
Size (KiB): 2368
Size (%): 3
Efficiency (%): 85
Sequentiality (%): 37.8

Entity: moz_places_hostindex
Count: -
Size (KiB): 4096
Size (%): 5.3
Efficiency (%): 84.9
Sequentiality (%): 37.8

Entity: moz_places_visitcount
Count: -
Size (KiB): 1344
Size (%): 1.7
Efficiency (%): 91.9
Sequentiality (%): 37.8

Entity: moz_places_frecencyindex
Count: -
Size (KiB): 1600
Size (%): 2.1
Efficiency (%): 84.5
Sequentiality (%): 37.8

Entity: moz_places_lastvisitdateindex
Count: -
Size (KiB): 2432
Size (%): 3.1
Efficiency (%): 93.7
Sequentiality (%): 37.8

Entity: moz_places_guid_uniqueindex
Count: -
Size (KiB): 3296
Size (%): 4.2
Efficiency (%): 85.5
Sequentiality (%): 37.8

Entity: moz_places_originidindex
Count: -
Size (KiB): 1760
Size (%): 2.3
Efficiency (%): 82.2
Sequentiality (%): 37.8

Entity: moz_historyvisits_placedateindex
Count: -
Size (KiB): 6304
Size (%): 8.1
Efficiency (%): 81.6
Sequentiality (%): 37.8

Entity: moz_historyvisits_fromindex
Count: -
Size (KiB): 2944
Size (%): 3.8
Efficiency (%): 93.9
Sequentiality (%): 37.8

Entity: moz_historyvisits_dateindex
Count: -
Size (KiB): 4352
Size (%): 5.6
Efficiency (%): 96.2
Sequentiality (%): 37.8

Entity: moz_bookmarks_itemindex
Count: -
Size (KiB): 32
Size (%): 0
Efficiency (%): 18.4
Sequentiality (%): 37.8

Entity: moz_bookmarks_parentindex
Count: -
Size (KiB): 32
Size (%): 0
Efficiency (%): 19.6
Sequentiality (%): 37.8

Entity: moz_bookmarks_itemlastmodifiedindex
Count: -
Size (KiB): 32
Size (%): 0
Efficiency (%): 32.4
Sequentiality (%): 37.8

Entity: moz_bookmarks_dateaddedindex
Count: -
Size (KiB): 32
Size (%): 0
Efficiency (%): 27.8
Sequentiality (%): 37.8

Entity: moz_bookmarks_guid_uniqueindex
Count: -
Size (KiB): 32
Size (%): 0
Efficiency (%): 34.8
Sequentiality (%): 37.8

Entity: moz_keywords_placepostdata_uniqueindex
Count: -
Size (KiB): 32
Size (%): 0
Efficiency (%): 0
Sequentiality (%): 37.8

Entity: moz_annos_placeattributeindex
Count: -
Size (KiB): 32
Size (%): 0
Efficiency (%): 5.6
Sequentiality (%): 37.8

Entity: moz_items_annos_itemattributeindex
Count: -
Size (KiB): 32
Size (%): 0
Efficiency (%): 0
Sequentiality (%): 37.8

Entity: moz_places_metadata_placecreated_uniqueindex
Count: -
Size (KiB): 32
Size (%): 0
Efficiency (%): 0
Sequentiality (%): 37.8

Entity: moz_places_metadata_referrerindex
Count: -
Size (KiB): 32
Size (%): 0
Efficiency (%): 0
Sequentiality (%): 37.8

Entity: moz_places_metadata_snapshots_pinnedindex
Count: -
Size (KiB): 32
Size (%): 0
Efficiency (%): 0
Sequentiality (%): 37.8

Entity: moz_places_metadata_snapshots_extra_typeindex
Count: -
Size (KiB): 32
Size (%): 0
Efficiency (%): 0
Sequentiality (%): 37.8

Accessibility
-------------

Activated: false
Prevent Accessibility: 0

Library Versions
----------------

NSPR
Expected minimum version: 4.35
Version in use: 4.35

NSS
Expected minimum version: 3.85
Version in use: 3.85

NSSSMIME
Expected minimum version: 3.85
Version in use: 3.85

NSSSSL
Expected minimum version: 3.85
Version in use: 3.85

NSSUTIL
Expected minimum version: 3.85
Version in use: 3.85

Sandbox
-------

Seccomp-BPF (System Call Filtering): true
Seccomp Thread Synchronization: true
User Namespaces: true
Content Process Sandboxing: true
Media Plugin Sandboxing: true
Content Process Sandbox Level: 4
Effective Content Process Sandbox Level: 4
Win32k Lockdown State for Content Process: Win32k Lockdown disabled -- Operating system not supported
GPU Process Sandbox Level: 0

Rejected System Calls
---------------------

Startup Cache
-------------

Disk Cache Path: /home/dusoft/.cache/mozilla/firefox/emol25ca.default-release/startupCache/startupCache.8.little
Ignore Disk Cache: false
Found Disk Cache on Init: false
Wrote to Disk Cache: true

Internationalization & Localization
-----------------------------------

Application Settings
Requested Locales: ["en-US"]
Available Locales: ["en-US","es-ES","es-CL","es-AR","es-MX"]
App Locales: ["en-US"]
Regional Preferences: ["en-US"]
Default Locale: "en-US"
Operating System
System Locales: ["en-US"]
Regional Preferences: ["sk-SK"]

Remote Debugging (Chromium Protocol)
------------------------------------

Accepting Connections: false
URL:

Printing
--------

Modified print settings
-----------------------

print_printer: Mozilla Save to PDF
print.more-settings.open: true
print.printer_Canon-MF4800-Series-UFRII-LT.print_bgcolor: false
print.printer_Canon-MF4800-Series-UFRII-LT.print_bgimages: false
print.printer_Canon-MF4800-Series-UFRII-LT.print_duplex: 0
print.printer_Canon-MF4800-Series-UFRII-LT.print_edge_bottom: 0
print.printer_Canon-MF4800-Series-UFRII-LT.print_edge_left: 0
print.printer_Canon-MF4800-Series-UFRII-LT.print_edge_right: 0
print.printer_Canon-MF4800-Series-UFRII-LT.print_edge_top: 0
print.printer_Canon-MF4800-Series-UFRII-LT.print_footercenter:
print.printer_Canon-MF4800-Series-UFRII-LT.print_footerleft: &PT
print.printer_Canon-MF4800-Series-UFRII-LT.print_footerright: &D
print.printer_Canon-MF4800-Series-UFRII-LT.print_headercenter:
print.printer_Canon-MF4800-Series-UFRII-LT.print_headerleft: &T
print.printer_Canon-MF4800-Series-UFRII-LT.print_headerright: &U
print.printer_Canon-MF4800-Series-UFRII-LT.print_in_color: false
print.printer_Canon-MF4800-Series-UFRII-LT.print_margin_bottom: 0.5
print.printer_Canon-MF4800-Series-UFRII-LT.print_margin_left: 0.5
print.printer_Canon-MF4800-Series-UFRII-LT.print_margin_right: 0.5
print.printer_Canon-MF4800-Series-UFRII-LT.print_margin_top: 0.5
print.printer_Canon-MF4800-Series-UFRII-LT.print_orientation: 0
print.printer_Canon-MF4800-Series-UFRII-LT.print_page_delay: 50
print.printer_Canon-MF4800-Series-UFRII-LT.print_paper_height: 11.6929
print.printer_Canon-MF4800-Series-UFRII-LT.print_paper_id: iso_a4_210x297mm
print.printer_Canon-MF4800-Series-UFRII-LT.print_paper_size_unit: 0
print.printer_Canon-MF4800-Series-UFRII-LT.print_paper_width: 8.26811
print.printer_Canon-MF4800-Series-UFRII-LT.print_resolution: 0
print.printer_Canon-MF4800-Series-UFRII-LT.print_reversed: false
print.printer_Canon-MF4800-Series-UFRII-LT.print_scaling: 1
print.printer_Canon-MF4800-Series-UFRII-LT.print_shrink_to_fit: true
print.printer_Canon-MF4800-Series-UFRII-LT.print_to_file: false
print.printer_Canon-MF4800-Series-UFRII-LT.print_unwriteable_margin_bottom: 20
print.printer_Canon-MF4800-Series-UFRII-LT.print_unwriteable_margin_left: 20
print.printer_Canon-MF4800-Series-UFRII-LT.print_unwriteable_margin_right: 20
print.printer_Canon-MF4800-Series-UFRII-LT.print_unwriteable_margin_top: 20
print.printer_CLP-320-Series.print_bgcolor: false
print.printer_CLP-320-Series.print_bgimages: false
print.printer_CLP-320-Series.print_duplex: 0
print.printer_CLP-320-Series.print_edge_bottom: 0
print.printer_CLP-320-Series.print_edge_left: 0
print.printer_CLP-320-Series.print_edge_right: 0
print.printer_CLP-320-Series.print_edge_top: 0
print.printer_CLP-320-Series.print_footercenter:
print.printer_CLP-320-Series.print_footerleft: &PT
print.printer_CLP-320-Series.print_footerright: &D
print.printer_CLP-320-Series.print_headercenter:
print.printer_CLP-320-Series.print_headerleft: &T
print.printer_CLP-320-Series.print_headerright: &U
print.printer_CLP-320-Series.print_in_color: true
print.printer_CLP-320-Series.print_margin_bottom: 0.5
print.printer_CLP-320-Series.print_margin_left: 0.5
print.printer_CLP-320-Series.print_margin_right: 0.5
print.printer_CLP-320-Series.print_margin_top: 0.5
print.printer_CLP-320-Series.print_orientation: 0
print.printer_CLP-320-Series.print_page_delay: 50
print.printer_CLP-320-Series.print_paper_height: 11.6945
print.printer_CLP-320-Series.print_paper_id: iso_a4_210x297mm
print.printer_CLP-320-Series.print_paper_size_unit: 0
print.printer_CLP-320-Series.print_paper_width: 8.26378
print.printer_CLP-320-Series.print_resolution: 0
print.printer_CLP-320-Series.print_reversed: false
print.printer_CLP-320-Series.print_scaling: 1
print.printer_CLP-320-Series.print_shrink_to_fit: true
print.printer_CLP-320-Series.print_to_file: false
print.printer_CLP-320-Series.print_unwriteable_margin_bottom: 17
print.printer_CLP-320-Series.print_unwriteable_margin_left: 13
print.printer_CLP-320-Series.print_unwriteable_margin_right: 13
print.printer_CLP-320-Series.print_unwriteable_margin_top: 17
print.printer_Mozilla_Save_to_PDF.print_bgcolor: false
print.printer_Mozilla_Save_to_PDF.print_bgimages: false
print.printer_Mozilla_Save_to_PDF.print_duplex: 0
print.printer_Mozilla_Save_to_PDF.print_edge_bottom: 0
print.printer_Mozilla_Save_to_PDF.print_edge_left: 0
print.printer_Mozilla_Save_to_PDF.print_edge_right: 0
print.printer_Mozilla_Save_to_PDF.print_edge_top: 0
print.printer_Mozilla_Save_to_PDF.print_footercenter:
print.printer_Mozilla_Save_to_PDF.print_footerleft: &PT
print.printer_Mozilla_Save_to_PDF.print_footerright: &D
print.printer_Mozilla_Save_to_PDF.print_headercenter:
print.printer_Mozilla_Save_to_PDF.print_headerleft: &T
print.printer_Mozilla_Save_to_PDF.print_headerright: &U
print.printer_Mozilla_Save_to_PDF.print_in_color: true
print.printer_Mozilla_Save_to_PDF.print_margin_bottom: 0.5
print.printer_Mozilla_Save_to_PDF.print_margin_left: 0.5
print.printer_Mozilla_Save_to_PDF.print_margin_right: 0.5
print.printer_Mozilla_Save_to_PDF.print_margin_top: 0.5
print.printer_Mozilla_Save_to_PDF.print_orientation: 0
print.printer_Mozilla_Save_to_PDF.print_page_delay: 50
print.printer_Mozilla_Save_to_PDF.print_paper_height: 11.6929133858268
print.printer_Mozilla_Save_to_PDF.print_paper_id: iso_a4
print.printer_Mozilla_Save_to_PDF.print_paper_size_unit: 0
print.printer_Mozilla_Save_to_PDF.print_paper_width: 8.26771653543307
print.printer_Mozilla_Save_to_PDF.print_reversed: false
print.printer_Mozilla_Save_to_PDF.print_scaling: 1
print.printer_Mozilla_Save_to_PDF.print_shrink_to_fit: true
print.printer_Mozilla_Save_to_PDF.print_to_file: true
print.printer_Mozilla_Save_to_PDF.print_unwriteable_margin_bottom_twips: 0
print.printer_Mozilla_Save_to_PDF.print_unwriteable_margin_left_twips: 0
print.printer_Mozilla_Save_to_PDF.print_unwriteable_margin_right_twips: 0
print.printer_Mozilla_Save_to_PDF.print_unwriteable_margin_top_twips: 0
print.printer_Print_to_File.print_bgcolor: false
print.printer_Print_to_File.print_bgimages: false
print.printer_Print_to_File.print_duplex: 0
print.printer_Print_to_File.print_edge_bottom: 0
print.printer_Print_to_File.print_edge_left: 0
print.printer_Print_to_File.print_edge_right: 0
print.printer_Print_to_File.print_edge_top: 0
print.printer_Print_to_File.print_footercenter:
print.printer_Print_to_File.print_footerleft: &PT
print.printer_Print_to_File.print_footerright: &D
print.printer_Print_to_File.print_headercenter:
print.printer_Print_to_File.print_headerleft: &T
print.printer_Print_to_File.print_headerright: &U
print.printer_Print_to_File.print_in_color: true
print.printer_Print_to_File.print_margin_bottom: 0.5
print.printer_Print_to_File.print_margin_left: 0.5
print.printer_Print_to_File.print_margin_right: 0.5
print.printer_Print_to_File.print_margin_top: 0.5
print.printer_Print_to_File.print_orientation: 0
print.printer_Print_to_File.print_page_delay: 50
print.printer_Print_to_File.print_paper_height: 11.6929
print.printer_Print_to_File.print_paper_id: iso_a4
print.printer_Print_to_File.print_paper_size_unit: 0
print.printer_Print_to_File.print_paper_width: 8.26772
print.printer_Print_to_File.print_reversed: false
print.printer_Print_to_File.print_scaling: 1
print.printer_Print_to_File.print_shrink_to_fit: true
print.printer_Print_to_File.print_to_file: false
print.printer_Print_to_File.print_unwriteable_margin_bottom: 0
print.printer_Print_to_File.print_unwriteable_margin_left: 0
print.printer_Print_to_File.print_unwriteable_margin_right: 0
print.printer_Print_to_File.print_unwriteable_margin_top: 0
```

**Comment by dannycolin** [comment]
That's expected and also happens with regular website that aren't assigned with a container. We do have a workaround that you can enable in the addon preferences for assigned websites. However, the same workaround hasn't been implemented for unassigned websites. 

I'm going to close this issue in favor of a previous report asking to implement exactly this feature.

**Comment by dannycolin** [comment]
Duplicate of #1852

**Comment by nekromoff** [comment]
There seems to be a misunderstanding as I checked the supposed duplicate, but this is a different bug.

What I have reported used to work before. Now it doesn't.

See this GIF recording (I will delete it once you confirm you can see the bug):
https://www.ambience.sk/bug.gif

**Comment by dannycolin** [comment]
> See this GIF recording (I will delete it once you confirm you can see the bug):

This isn't a feature from Multi-Account Containers. It looks to be something from https://addons.mozilla.org/en-US/firefox/addon/context-plus/.

**Comment by nekromoff** [comment]
Ah, well spotted, thanks! Sorry.
